engine/src/exec-pasteboard.cpp

```diff
@@ -279,7 +279,7 @@ MCTransferType MCPasteboardFindTransferTypeForLegacyClipboard(const MCClipboard*
     // passed that the behaviour can now be corrected but, until then, it is
     // disabled and the "text over images" method is used instead.
     //
-    if (false)
+    if (/* DISABLES CODE */ (false))
```

Comment:
Could you please add something about using `/* DISABLES CODE */ (...)` in this way to `CONTRIBUTING.md`?  It's similar to our use of `/* UNCHECKED */` in many ways and if it's done in a consistent way then it becomes similarly greppable.